wikibase.api.js

```diff
@@ -13,7 +13,7 @@ class WikibaseApi {
 	 */
 	createItem( label, data ) {
```

Comment:
So you are issuing requests without a label but with data?

Reply:
Well, some tests create entirely empty items (like the termbox tests).